client/login/magic-login/style.scss

```diff
@@ -242,10 +242,10 @@
 
 .magic-login__form-action {
 	margin-top: 20px;
-	overflow: auto;
 
 	button {
 		width: 100%;
+		justify-content: center;
```

Comment:
Curious why this was necessary, is this one of the button components?

Reply:
I've found these styles have been necessary in all the button replacements so far. The text ends up being aligned left without. I think it's because the core buttons by default are `display: inline-flex; width: initial`. `justify-content: center` would seem like a smart default for them imo.

With these styles toggled off:

![Screenshot 2025-05-07 at 2 11 14 PM](https://github.com/user-attachments/assets/54f7e49d-267e-4450-bfcc-056a25368012)

Reply:
CC: @ciampo @mirka

Reply:
What Marco says 💯  We need to be careful to keep everything consistent with core. If we want to change this for all `Button`s, OK, but that means forking `Button` locally and applying changes in `@automattic/components`. Is this something we all agree we want to diverge from the WP core on? IMO the direction here needs to come from the design team after agreeing on the next step.

Reply:
Noting that we have some overrides configured [here](https://github.com/Automattic/wp-calypso/blob/0a41bf560cfd7666292cda907a887c4b7d35029f/client/assets/stylesheets/_wp-components-overrides.scss#L8), which I have been tempted to add this rule to. Could be an interim solution to avoid peppering the rule around the codebase, rather than forking?

Reply:
I think it would be useful to have a global override for that. I also [added an instance](https://github.com/Automattic/wp-calypso/pull/102616/files#diff-83011c4eb6b72241c1e520de9b2ff8a51b1784078661fbf25a067b63997455aaR145) of `justify-content: center` recently 😅 

To be fair, I think the Core component should have had that style to begin with. It's being overridden all over the place in Gutenberg too.

Reply:
For now, I created [an issue to track this](https://linear.app/a8c/issue/DS-171/button-add-justify-content-center).

Reply:
I'll add the override then so at least our interim fix is centralized, and can be easily removed once we have a fork or upstream fix.

Reply:
Done in [fb330f6](https://github.com/Automattic/wp-calypso/pull/103142/commits/fb330f66a2d1e59bee580dc889b1cae3b84b012e)